src/hooks/Trades.ts

```diff
@@ -1,5 +1,5 @@
 import { Currency, CurrencyAmount, Pair, Token, Trade } from '@dynamic-amm/sdk'
-import { useMemo, useEffect, useState, useCallback } from 'react'
+import { useMemo, useEffect, useState, useCallback, useReducer } from 'react'
```

Comment:
It's seem we do not use `useReducer` any where?

Reply:
agree, removed